Purchases/Public/RCPurchases.h

```diff
@@ -161,6 +172,48 @@ typedef NS_ENUM(NSInteger, RCAttributionNetwork) {
 /// Default to YES, set this to NO if you are finishing transactions with your own StoreKit queue listener
 @property (nonatomic) BOOL finishTransactions;
 
+/// This version of the Purchases framework
++ (NSString *)frameworkVersion;
+
+/**
+ Delegate for `RCPurchases` instance. The delegate is responsible for handling completed purchases and updated purchaser information.
+ 
+ @note `RCPurchases` will not listen for any purchases until the delegate is set. This prevents transactions from being processed before your app is ready to handle them.
+ */
+@property (nonatomic, weak) id<RCPurchasesDelegate> _Nullable delegate;
+
+#pragma mark Identity
+
+/// The `appUserID` used by `RCPurchases`. If not passed on initialization this will be generated and cached by `RCPurchases`.
+@property (nonatomic, readonly) NSString *appUserID;
+
+/**
+ This function will alias two appUserIDs together.
+ @param alias The new appUserID that should be linked to the currently identified appUserID
+ */
+- (void)createAlias:(NSString *)alias;
+
+/**
+ This function will alias two appUserIDs together.
+ @param alias The new appUserID that should be linked to the currently identified appUserID
+ @param completion An optional completion block called when the aliasing has been successful. This completion block will receive an error if there's been one.
+ */
+- (void)createAlias:(NSString *)alias completionBlock:(RCReceivePurchaserInfoBlock)completion
+NS_SWIFT_NAME(createAlias(_:_:));
+
+/**
+ This function will identify the current user with an appUserID. Typically this would be used after a logout to identify a new user without calling configure
+ @param appUserID The appUserID that should be linked to the currently user
+ */
+- (void)identify:(NSString * _Nullable)appUserID completionBlock:(RCReceivePurchaserInfoBlock _Nullable)completion
+NS_SWIFT_NAME(identify(_:_:));
```

Comment:
I think we could do the same as with createAlias, where the completion block is optional.